cli/failover/set.go

```diff
@@ -0,0 +1,79 @@
+package failover
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+func Set(ctx *context.Ctx) error {
+	if (ctx.Failover.Mode == "eventual" || ctx.Failover.Mode == "disabled") && ctx.Failover.ProviderParamsJSON != "" {
+		return fmt.Errorf("Please, don't specify provider parameters when using %s mode", ctx.Failover.Mode)
+	}
+
+	if err := FillCtx(ctx); err != nil {
+		return err
+	}
+
+	failoverOpts, err := getFailoverOpts(ctx)
+	if err != nil {
+		return err
+	}
+
+	log.Infof("Set up %s failover", failoverOpts.Mode)
```

Comment:
`Set up disabled failover`? Looks weird. What about calling `Disable` function for such a case? 

BTW, maybe `Configure <mode> failover`?

Reply:
If we call the ``Disable`` function, we will lose the `` failoverOpts`', for example is ``failover_timeout`` parameter - cartridge applies these parameters anyway, even if we pass mode ``disabled``. I think if we ignore these parameters it will be incorrect.

Reply:
Given that the parameters are applied when disabled, we can write <configure>. But, probably, I agree, that it can confuse. The simplest solution is to just ``log.Infof`` different lines depending on the opts.Mode

Reply:
We can do this for the ``set`` command, but not for the ``setup`` command (see https://github.com/tarantool/cartridge-cli/blob/b3b3469ea13e220d4e1065954bcb7ff89b8513d7/cli/failover/setup.go#L70)